packages/tocco-ui/src/EditableValue/typeEditors/DateAbstract.js

```diff
@@ -115,7 +115,9 @@ class DateAbstract extends React.Component {
   handleOnChange = selectedDates => {
     const isoStrings = selectedDates.map(date => date.toISOString())
     this.setState({altInput: this.flatpickr.altInput.value})
-    this.props.onChange(isoStrings)
+    if (!(!this.hasValue() && isoStrings.length === 0)) {
```

Comment:
Maybe simplify to `this.hasValue() || isoStrings.length !== 0`, which should be the same? would be easier to read at least for me.